featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -7,7 +7,16 @@
 from woodwork.column_schema import ColumnSchema
 from woodwork.logical_types import Boolean, Datetime, Integer, Ordinal
 
-import featuretools as ft
+from featuretools import (
+    AggregationFeature,
+    EntitySet,
+    Feature,
+    IdentityFeature,
+    TransformFeature,
+    calculate_feature_matrix,
+    dfs,
+    primitives,
```

Comment:
Import specific primitives instead of module.

Reply:
We access `dir(primitives)` in this file, is it ok to leave 'primitives' in the import statement?

Reply:
I didn't catch that. In that case, yes, it is fine to leave the import, but would confirm we aren't accessing any primitives with something like `primitives.Year`.